lisa/node.py

```diff
@@ -733,6 +748,24 @@ def check_sudo_password_required(self) -> None:
                     raise RequireUserPasswordException("Reset password failed")
             self._check_password_and_store_prompt()
 
+    def _collect_logs_using_non_ssh_executor(self) -> Optional[str]:
+        """
+        Collects information using the NonSshExecutor feature.
+        This is used when the connection to the node is not stable.
+        """
+        from lisa.features import NonSshExecutor
+
+        if self.features.is_supported(NonSshExecutor):
+            non_ssh_executor = self.features[NonSshExecutor]
+            out = non_ssh_executor.execute()
+            return "\n".join(out)
```

Comment:
Log could be output in this method directly, instead of return and output in upper level.